src/chatdDb.h

```diff
@@ -337,6 +337,12 @@ class ChatdSqliteDb: public chatd::DbInterface
         stmt.blobCol(2, buf);
         msg.assign(buf, stmt.intCol(0), stmt.uint64Col(3), stmt.intCol(1), stmt.uint64Col(4));
     }
+
+    virtual void clearHistory()
+    {
+        mDb.query("delete from history where chatid = ?", mMessages.chatId());
+        mDb.query("delete from chat_vars where chatid = ?", mMessages.chatId());
```

Comment:
Despite currently the `chat_vars` table only stores rows with the `name='have_all_history'`, I think it's more correct to modify this sql sentence to `delete from chat_vars where chatid = ? and name='have_all_history'`